src/geometry/ArborX_GeometryTraits.hpp

```diff
@@ -62,13 +62,15 @@ DEFINE_GEOMETRY(sphere, SphereTag);
 DEFINE_GEOMETRY(triangle, TriangleTag);
 DEFINE_GEOMETRY(kdop, KDOPTag);
 DEFINE_GEOMETRY(ray, RayTag);
+DEFINE_GEOMETRY(tetrahedron, TetrahedronTag);
 #undef DEFINE_GEOMETRY
 
 template <typename Geometry>
 inline constexpr bool
     is_valid_geometry = (is_point_v<Geometry> || is_box_v<Geometry> ||
                          is_sphere_v<Geometry> || is_kdop_v<Geometry> ||
-                         is_triangle_v<Geometry> || is_ray_v<Geometry>);
+                         is_triangle_v<Geometry> || is_ray_v<Geometry> ||
+                         is_tetrahedron_v<Geometry>);
```

Comment:
Nitpick: I'd prefer to list ray last to segregate it as I see it as an odd geometry type.